JetNews/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  ci:
+    name: Build + Test
+    runs-on: macOS-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```

Comment:
It's good idea to use `fetch-depth: 1` here, especially for repos which uses contain prebuilts (or have previously contained prebuilts).

Reply:
`fetch-depth: 1` is the default value but can add it (just in case in changes in the future?). Thanks!

Reply:
Ah, TIL 👍